packages/auto-chart/src/AutoChart.tsx

```diff
@@ -1,11 +1,17 @@
-import React from 'react';
+import React, { useState, useRef, useEffect } from 'react';
+import { Advisor } from '@antv/chart-advisor';
+import { Chart } from './ChartRender';
+import { AdviceList } from './AdviceList';
+import { ChartConfigPanel, ChartConfigBtn } from './ChartConfigPanel';
+import { prefixCls } from './utils';
+import './index.less';
 
 interface Props {
-  id: string;
   className?: string;
   width?: number;
   height?: number;
   data?: any[];
+  fields?: any[];
```

Comment:
```suggestion
  fields?: string[];
```

这个 props 配置与 chart-advisor 一致即可

Reply:
已经修改